src/yunohost/service.py

```diff
@@ -296,10 +296,10 @@ def service_status(names=[]):
         # the hack was to add fake services...
         # we need to extract regenconf from service at some point, also because
         # some app would really like to use it
-        if services[name].get("status", "") is None:
+        if infos.get("status", "") is None:
```

Comment:
I'm confused:

```suggestion
        if infos.get("status") is None:
```

Reply:
Eeeeeh there's a not-obvious subtlety there ... Most services don't have this "status" key ... but some do have `"status" = None` and these are the one we want to catch with that if. Maybe we could have `if "status" in infos and infos["status"] is None` ... but that doesn't make it more explicit. We could also improve the comment..

Reply:
I understand, sorry for the noise.